medusa/session/hooks.py

```diff
@@ -28,8 +28,8 @@ def log_url(response, **kwargs):
 
     if request.method.upper() == 'POST':
         body = request.body \
-            if 'multipart/form-data' not in request.headers.get('content-type') \
-            else request.body[1:99].replace('\n', ' ') + '...'
+            if 'multipart/form-data' not in request.headers.get('content-type', '') \
+            else request.body[1:99].replace('\n', ' ') + '...' if request.body else None
```

Comment:
better use `''` instead of None, as where decoding a few lines down. And None.decode('utf-8') will cause an exception.